lib/api/controllers/documentController.js

```diff
@@ -72,11 +73,14 @@ function DocumentController (kuzzle) {
     assertBodyAttributeType(request, 'ids', 'array');
     assertHasIndexAndCollection(request);
 
+    if (request.input.body.ids.length > kuzzle.config.server.maxMultiActionsCount) {
```

Comment:
You can also make a search request with a from / size resulting to a batch of 9 999 documents (perhaps 10 000).